src/server.ts

```diff
@@ -19,18 +19,78 @@ app
         .status(StatusCodes.BAD_REQUEST)
         .json({ error: "playerId is invalid" });
 
-    const player = await prisma.player.findUnique({ where: { playerId } });
-
-    const { greenboxString, greenboxLastUpdate } = player ?? {};
+    const latestGreenbox = await prisma.greenbox.findFirst({
+      where: { playerId },
+      orderBy: { id: "desc" },
+      select: {
+        player: true,
+        data: true,
+        createdAt: true,
+      },
+    });
 
-    if (!greenboxString || !greenboxLastUpdate)
+    if (!latestGreenbox)
       return res
         .status(StatusCodes.NOT_FOUND)
         .json({ error: "No greenbox data found" });
 
+    const count = await prisma.greenbox.count({
+      where: { playerId },
+    });
+
+    return res.status(StatusCodes.OK).json({
+      data: latestGreenbox.data,
+      createdAt: latestGreenbox.createdAt,
+      count,
+    });
+  })
+  .get("/api/greenbox/:playerId/:greenboxNumber", async (req, res) => {
```

Comment:
Oh neat, I wasn't sure whether the overlap with `"/api/greenbox/:playerId"` would cause issues. Good to know it won't.